var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -40,6 +41,7 @@ class Rust(Package):
     depends_on("rust-bootstrap@1.59:1.60", type="build", when="@1.60")
     depends_on("rust-bootstrap@1.64:1.65", type="build", when="@1.65")
     depends_on("rust-bootstrap@1.69:1.70", type="build", when="@1.70")
+    depends_on("rust-bootstrap@1.73:1.74", type="build", when="@1.73")
```

Comment:
```suggestion
    depends_on("rust-bootstrap@1.72:1.73", type="build", when="@1.73")
```

Can you not bootstrap rust@1.73 with the previous 1.72?

Reply:
I believe you can, but when I added the bootstrap version I grabbed the same. Makes sense to expand it, good idea.